src/error.rs

```diff
@@ -233,6 +234,11 @@ impl fmt::Display for PsbtValidationError {
                 "This input refers to an output of invalid type: '{:#?}'",
                 i
             ),
+            Self::InvalidCountOuputWithDerivations(c) => write!(
+                f,
+                "An incorrect number of PSBT outputs had the bip32 derivation set. Expected {}.",
```

Comment:
giant nit, you could say how many were provided

Reply:
Meh :(

Reply:
Actually if we do that we should as well put the min.. And that becomes dirty. Instead i just replaced expected by the actual invalid count.